src/diffusers/models/cross_attention.py

```diff
@@ -87,8 +90,27 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
 
+class TuneAVideoCrossAttnProcessor(TuneAVideoAttnProcessor):
+    def __init__(self, *args, **kwargs):
+        deprecation_message = f"{self.__class__.__name__} is deprecated and will be removed in `0.18.0`. Please use `from diffusers.models.attention_processor import {''.join(self.__class__.__name__.split('Cross'))} instead."
+        deprecate("cross_attention", "0.18.0", deprecation_message, standard_warn=False)
+        super().__init__(*args, **kwargs)
+
+
 class SlicedCrossAttnAddedKVProcessor(SlicedAttnAddedKVProcessor):
     def __init__(self, *args, **kwargs):
         deprecation_message = f"{self.__class__.__name__} is deprecated and will be removed in `0.18.0`. Please use `from diffusers.models.attention_processor import {''.join(self.__class__.__name__.split('Cross'))} instead."
         deprecate("cross_attention", "0.18.0", deprecation_message, standard_warn=False)
         super().__init__(*args, **kwargs)
+
+
+AttnProcessor = Union[
```

Comment:
We can remove all this I think - note that the file `cross_attention.py` is outdated

Reply:
Take note of this ^.

Reply:
Let's not do any changes to this file, this file is deprecated

Reply:
Removed as explained above.